adage/analyze/api.py

```diff
@@ -275,12 +275,37 @@ class Meta:
         queryset = Node.objects.all()
         resource_name = 'node'
         allowed_methods = ['get']
+        multiple_allowed_methods = ['post']
```

Comment:
Hmmm, I haven't been able to find any docs on this Meta ``multiple_allowed_methods`` attribute. Is it a custom one you made for this functionality?